<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="usernameInput">
     <property name="placeholderText">
      <string>Логин</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="passwordInput">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
     </property>
     <property name="placeholderText">
      <string>Пароль</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>382</width>
       <height>162</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>382</width>
       <height>162</height>
      </size>
     </property>
     <property name="title">
      <string>Настройки сервера</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="hostInput">
        <property name="placeholderText">
         <string>Имя хоста</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="portInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Подключиться</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="registerButton">
     <property name="text">
      <string>Регистрация</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="loginButton">
     <property name="text">
      <string>Вход</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
